src/types.js

```diff
@@ -34,8 +34,14 @@ export const POP = 'ROUTER_POP';
 export const REPLACE_ROUTES = 'ROUTER_REPLACE_ROUTES';
 export const DID_REPLACE_ROUTES = 'ROUTER_DID_REPLACE_ROUTES';
 
+const actionsWithPayload = [PUSH, REPLACE, GO, POP];
+const actions = [...actionsWithPayload, GO_FORWARD, GO_BACK, POP];
```

Comment:
This'll unfortunately require `Symbol` polyfilling 😢 let's just concat?

Reply:
@tptee the `lib` and `dist` builds both compile this to `concat` so that shouldn't matter right?

Reply:
I’m dumb, it’s array destructuring that requires symbol. Carry on